client/state/account-recovery/reset/reducer.js

```diff
@@ -17,20 +17,44 @@ import {
 	ACCOUNT_RECOVERY_RESET_REQUEST_ERROR,
 	ACCOUNT_RECOVERY_RESET_UPDATE_USER_DATA,
 	ACCOUNT_RECOVERY_RESET_PICK_METHOD,
+	ACCOUNT_RECOVERY_RESET_SET_VALIDATION_KEY,
+	ACCOUNT_RECOVERY_RESET_VALIDATE_REQUEST,
+	ACCOUNT_RECOVERY_RESET_VALIDATE_REQUEST_SUCCESS,
+	ACCOUNT_RECOVERY_RESET_VALIDATE_REQUEST_ERROR,
 } from 'state/action-types';
 
+const createRequestingReducer = ( onActions, offActions ) => createReducer( false, {
```

Comment:
I think you are right. If I really wanted to introduce it, it might better be in `state/utils.js`. Even so, I think making another PR for it would be better. From what I'm seeing now, removing the helper functions and using `stubTrue` and `stubFalse` should be enough for this PR: ea7cc08b3c72f10e8b8bbd3d6acb9542dd4cf312.